app/customer.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+
+from app.car import Car
+from app.shop import Shop
+
+
+@dataclass
+class Customer:
+    name: str
+    location: list
+    money: int or float
```

Comment:
The type hint `int or float` is not standard. It should be `Union[int, float]` from the `typing` module if you want to specify multiple types.